src/containers/homepage/survey-list.component.tsx

```diff
@@ -62,6 +84,64 @@ export class SurveyListComponent extends Component<SurveyListComponentProps> {
                 ),
             },
         ];
-        return <Table dataSource={items} columns={columns} bordered />;
+
+        return (
+            <>
+                <Row type="flex" justify="space-between">
+                    <Col span={12}>
+                        <h2>Surveys</h2>
+                    </Col>
+                    <Col span={12}>
+                        <Link to="/survey/new">
+                            <Tooltip placement="left" title="Create a new survey">
+                                <Button type="primary" style={{ float: 'right' }}>
+                                    <Icon type="plus" />
+                                    New survey
+                                </Button>
+                            </Tooltip>
+                        </Link>
+                    </Col>
+                </Row>
+                <Table
+                    dataSource={this.filter(items)}
+                    columns={columns}
+                    bordered
+                    title={() => (
```

Comment:
Q: Does this need to be a function? Have you tried just passing an object here?

Reply:
Just checked antd, this does indeed have to be a function - could be a property instead but it seems like premature optimization